### ISSUE TITLE
Coding: Add more information to the Introduction Page

### ISSUE THREAD

**Issue by anhushree** [kothariji/SyntaxMeets#49]
**Is your feature request related to a problem? Please describe.**
The Intro page (Landing Page) misses  FAQs, Contact Us for Queries slides.
A User who first sees SyntaxMeets will find it easier to get to know it better.

**Describe the solution you'd like**
We can add more sections on the Intro Page.
As the user scrolls down he can find the FAQs, Contact us slides
Optional: Can include the About Us section in the Intro Page itself.

**Motivation**
Please outline the motivation for the proposal.
I've seen other Landing pages as simple as this but offer information to the user.

**Are you willing to contribute to this issue?** [Yes/No]
Yes

**Comment by Akash-Salvi** [comment]
Hi @anhushree, First of all, thank you so much for taking out the time to create this issue.

It will be very helpful for others to get more information about SyntaxMeets.
Also, adding GitHub and About Us buttons on the home page would increase user experience overall.
But before working on the actual code-part it would be better if you could share with us a basic Figma/xd/canva/excalidraw prototype design of the home page so that we can tell if there are any changes.
Definitely 🙌, Please go forward to work on this issue. I am assigning this Issue to you 👍

And last but not least,
Keep Contributing 🙏🔥

**Comment by anhushree** [comment]
Sure will share the UI design first, Thank you.

**Comment by anhushree** [comment]
https://www.figma.com/file/ZHGDWGlGruVU8JZ3cJBX7p/SyntaxMeets-UI?node-id=0%3A1
@Akash-Salvi Can you check this? If any changes do let me know. This is just a sample for reference.

**Comment by kothariji** [comment]
any updates @anhushree  ?

**Comment by anhushree** [comment]
> any updates @anhushree?
Yes, I'm working on it well regarding the design I have few suggestions. I'll share the design page after updating it by tonight.

**Comment by anhushree** [comment]
https://www.figma.com/file/ZHGDWGlGruVU8JZ3cJBX7p/SyntaxMeets-UI?node-id=0%3A1 @kothariji Kindly check this

**Comment by anhushree** [comment]
Is there anything to be done on the UI improvement part? Kindly let me know @kothariji

**Comment by kothariji** [comment]
actually we are not in any plans for changing it, so let's keep that on hold, and once we feel like changing , we will definitely inform you :raised_hands:
